### PR TITLE
add images for version 6.4.3.0

### PR DESCRIPTION
**Pull request by bpesch**

### CODE REVIEW

File: variants/dev/6.4.3.0/variables.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "shopware": {
+    "env": "dev",
+    "version": "6.4.3.0",
+    "download_url": "https://www.shopware.com/en/Download/redirect/version/sw6/file/install_v6.4.2.1_eccc23da5632314390e2a3715afb72ab0a6d5ab9.zip"
```
Comment (martinweinmayr) [inline variants/dev/6.4.3.0/variables.json:5]: pls change download link to: https://www.shopware.com/de/Download/redirect/version/sw6/file/install_v6.4.3.0_1783a8a29c331b9568274c727b3dae5f73ba142a.zip

File: variants/play/6.4.3.0/variables.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "shopware": {
+    "version": "6.4.3.0",
+    "download_url": "https://www.shopware.com/en/Download/redirect/version/sw6/file/install_v6.4.2.1_eccc23da5632314390e2a3715afb72ab0a6d5ab9.zip"
```
Comment (martinweinmayr) [inline variants/play/6.4.3.0/variables.json:4]: pls change download link to https://www.shopware.com/de/Download/redirect/version/sw6/file/install_v6.4.3.0_1783a8a29c331b9568274c727b3dae5f73ba142a.zip

**Review by martinweinmayr — CHANGES_REQUESTED** [review]
many thanks for you PR! Xou are the first one that creats a new version within dockware woooho.

If you fix the 2 issues, i will approve and build the versions

**Review by martinweinmayr — APPROVED** [review]

File: variants/dev/6.4.3.0/variables.json (hunk 1 above)
Comment (bpesch) [reply variants/dev/6.4.3.0/variables.json]: my bad, fixed!

File: variants/play/6.4.3.0/variables.json (hunk 2 above)
Comment (bpesch) [reply variants/play/6.4.3.0/variables.json]: my bad, fixed!

**Pull request merged**